Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,190 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton.js';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
 
-const App = () => {
+class App extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { total: 0,
+                   displayNum: '0',
+                   currentOperator: '',
+                   opBtnStyle: 'red',
+                   opOne: '', //operandOne 
+                   aO: '' //activeOperator sign in display (plus, minus, etc.)
+                  };
+    this.aTDoC = this.aTDoC.bind(this);
+    this.clearDisplay = this.clearDisplay.bind(this);
+    this.activeOperator = this.activeOperator.bind(this);
+    this.calculate = this.calculate.bind(this);
+  }
+  /* add to display on click */
+  aTDoC(number){
+    let numberArr = this.state.displayNum;
+    if(this.state.displayNum === '0'){
+    numberArr = '';  
+    numberArr += number; 
+    this.setState({displayNum: numberArr})
+    }
+    else{
+      numberArr += number;
+      this.setState({displayNum: numberArr})
+    } 
+  }
+
+  clearDisplay(){
+    this.setState({displayNum: '0', opOne: '', aO: ''});
+  }
+
+  activeOperator(txt){
+
+    if(txt === this.state.aO){
+      this.setState({aO: '', opOne: ''})
+    }
+    else {
+      switch(txt){
+        case 'min': 
+        this.setState({aO: txt, opOne: this.state.displayNum});
+        this.state.displayNum = '0';
+        break;
+        case 'div': 
+        this.setState({aO: txt, opOne: this.state.displayNum});
+        this.state.displayNum = '0';
+        break;
+        case 'plus': 
+        this.setState({aO: txt, opOne: this.state.displayNum});
+        this.state.displayNum = '0';
+        break;
+        case 'mult': 
+        this.setState({aO: txt, opOne: this.state.displayNum});
+        this.state.displayNum = '0';
+        break;
+      }
+    }
+  }
+
+  calculate(op1,op2,opSign){
+    let total;
+    switch(opSign){
+      case 'min':
+        op1 = Number(op1);
+        op2 = Number(op2);
+        total = op1 - op2; 
+        total.toString();
+        this.setState({displayNum: total});
+        this.setState({aO: '', opOne: ''});
+        
+        break;
+        case 'div': 
+        op1 = Number(op1);
```

Comment:
you need to keep your indentation on point with stuff like this, otherwise it's very hard to read. Consider using Prettier.